io.catenax.shared.business_partner_certificate/2.0.0/BusinessPartnerCertificate.ttl

```diff
@@ -0,0 +1,173 @@
+   ] .
+	
+ :EnclosedSiteEntity a samm:Entity ;
+   samm:preferredName "Enclosed Site Entity"@en ;
+   samm:description "Entity representing an enclosed site, can be BPNS or BPNA"@en ;
+   samm:properties (:enclosedSiteBpn [samm:property :areaOfApplication; samm:optional true]) .
+
+  :enclosedSiteBpn a samm:Property ;
+   samm:preferredName "enclosedSite BPN"@en ;
+   samm:description "The BPN of an enclosed site"@en ;
+   samm:characteristic ext-number:BpnsTrait ;  # This can be set to handle BPNS and BPNA
+   samm:exampleValue "BPNS00000003AYRE" .
+ 
+  :validFrom a samm:Property;
+    samm:preferredName "Valid from"@en;
+    samm:description "Valid from date as defined on the certificate."@en;
+    samm:characteristic :Date;
+    samm:exampleValue "2023-01-25"^^xsd:date.
+
+ :validUntil a samm:Property;
+    samm:preferredName "Valid until"@en;
+    samm:description "Valid valid until as defined on the certificate. If certificate never expires value until expected to be 9999-12-31"@en;
+    samm:characteristic :Date;
+	samm:exampleValue "2026-01-24"^^xsd:date.
+    
+ :Date a samm:Characteristic ;
+   samm:preferredName "Date"@en ;
+   samm:description "Describes a property which contains the date in english format."@en ;
+   samm:dataType xsd:date;
+   samm:exampleValue "2026-01-24"^^xsd:date.
+  
+:validator a samm:Property ;
+   samm:preferredName "Validator"@en ;
+   samm:description "The BPN of the data service provider who validate the given certificate"@en ;
+   samm:characteristic :TrustValidatorCharacteristic.
+
+:TrustValidatorCharacteristic a samm:Characteristic;
+	samm:preferredName "Validiator caracteristic"@en;
+	samm:description "The BPN of the data service provider who validated the given certificate"@en ;
+    samm:dataType :TrustValidatorEntitity . 
+   
+   
+:TrustValidatorEntitity a samm:Entity ;
+   samm:preferredName "Trust validator entity"@en ;
+   samm:description "The BPN of the data service provider who validated the given certificate"@en ;
+   samm:properties ([samm:property :validatorName; samm:optional true] [samm:property :validatorBpn; samm:optional true]).
+
+:validatorName a samm:Property ;
+   samm:preferredName "Validator name"@en ;
+   samm:description "The optional name of the data service provider who validated the given certificate"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Data service provider X".
+
+:validatorBpn a samm:Property ;
+   samm:preferredName "Certifcate validator Bpn"@en ;
+   samm:description "The BPN of the data service provider who validated the given certificate"@en ;
+   samm:characteristic ext-number:BpnlTrait ;
+   samm:exampleValue "BPNL00000007YREZ".
+ 
+ :issuer a samm:Property ;
+   samm:preferredName "Issuing authority"@en ;
+   samm:description "The BPN of the issuing authority e.g. TUEV Sued "@en ;
+   samm:characteristic ext-number:BpnlTrait ;
+   samm:exampleValue "BPNL00000023ZAVC".
+
+ :trustLevel a samm:Property;
```

Comment:
please correct the alignment here

Reply:
Done, kindly have a review.

Reply:
what I mean is that below part should also be completely left aligned like others.
```
samm:preferredName "Trust level"@en ; 
samm:description "The trust level of the given certificate - none,low, high, trusted"@en ;
samm:characteristic :TrustLevelValue ;
samm:exampleValue "high".
```

Reply:
On visual studio code it was not showing alinement issue, but corrected it after rewriting the same.